contracts/sol6/Dao/MultipleEpochRewardsClaimer.sol

```diff
@@ -0,0 +1,48 @@
+pragma solidity 0.6.6;
+
+import "../IKyberDao.sol";
+import "./IMultipleEpochRewardsClaimer.sol";
+
+
+contract MultipleEpochRewardsClaimer is IMultipleEpochRewardsClaimer {
```

Comment:
lets add withdrawable 
to recover from user mistakes

Reply:
Updated